cmds/cpu: remove termios calls

rminnich opened the pull request:
Now that the u-root termios pkg is fixed as of b0da363052a2724a46a72ec11d323e6f646529a9,
we no longer need the calls in cpu.go to get and reset the terminal.

Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

rminnich committed:
cmds/cpu: remove termios calls in main()

Now that the u-root termios pkg is fixed as of b0da363052a2724a46a72ec11d323e6f646529a9,
we no longer need the calls in cpu.go to get and reset the terminal.

Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

rminnich committed:
client/client.go: if GetWinSize fails, mark the session non-interactive

If GetWinSize fails, we are not on a regular terminal. Mark the session
as not being interactive, so code will not try to call SetupInteractive()

Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

Lencerf commented on client/client.go:
```diff
@@ -100,7 +100,10 @@ func Command(host string, args ...string) *Cmd {
+		interactive = false
+		log.Printf("Can not get winsize: %v; assuming %dx%d and non-interactive", err, col, row)
 	} else {
 		col, row = int(w.Col), int(w.Row)
```
I think we need `interactive = true` here.

rminnich committed:
client/client.go: change interactive to hasTTY

The issue is not whether the program is run interactively, but
whether fd 0 is a tty or not.

The way to tell is to see if GetWinSize of fd 0 returns an error.
If it succeeds, use the returned size. If it fails, mark the
command as not having a TTY.

If there is no TTY, do not attach a remote PTY, and use io.Copy,
not TTYIn, to transfer the data.

These commands worked before:
cpu host
cpu host ls

These commands now work correctly and did not before:
cpu host bash
echo ls | host

Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>
